lib/mi_bridges/driver.rb

```diff
@@ -86,9 +87,22 @@ def run
 
     private
 
-    attr_reader :snap_application
+    def logger=(logger)
+      return @logger = logger if logger.present?
+      logger = ActiveSupport::Logger.new(STDOUT)
+      logger.level = if ENV["DEBUG_DRIVE"] = "true"
+                       Logger::DEBUG
+                     else
+                       Logger::INFO
+                     end
+      logger.formatter = Rails.application.config.log_formatter
+      @logger = ActiveSupport::TaggedLogging.new(logger)
+    end
+
+    attr_reader :snap_application, :logger
 
     def setup
+      # Do we really want to destroy all the stored driver applications?
       DriverApplication.destroy_all
```

Comment:
What do you think about - 

`DriverApplication.destroy_all if Rails.env.development?`

\- ?

Reply:
I think this is left over from before we removed it.